src/common/locales/strings/enUS.json

```diff
@@ -181,7 +181,7 @@
   "alert_scene_approve": "It was me, grant access",
   "alert_scene_deny": "Not me, deny all",
   "otp_scene_header_2fa": "This device cannot log in because it does not have the right 2-factor code",
-  "otp_scene_header_ip": "This device cannot log in due to an unrecognized IP address",
+  "otp_scene_header_ip": "Username and Password correct. This device cannot log in due to an unrecognized IP address.",
```

Comment:
We should lowercase "password", and maybe add the word "are" to make it a real sentence. "Username and password are correct."